brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/Alien4CloudFacade.java

```diff
@@ -414,7 +414,7 @@ private String buildExportStatements(String nodeId, Alien4CloudApplication tosca
         Optional<RelationshipTemplate> optionalRelationshipTemplate = findRelationshipRequirement(nodeId, toscaApplication, requirementId);
         if (optionalRelationshipTemplate.isPresent()) {
             RelationshipTemplate relationshipTemplate = optionalRelationshipTemplate.get();
-            if (relationshipTemplate.getType().equals("brooklyn.relationships.Configure")) {
+            if (relationshipTemplate.getType().equals("tosca.relationships.Configure")) {
```

Comment:
Switch this back and update `relationship.yaml` and `tomcat-mysql-on-one-compute.yaml` to use `brooklyn.relationships.Configure` instead.